app/helpers/view_helper.rb

```diff
@@ -143,9 +143,9 @@ def show_legacy_courses_table?
 
   def base_errors_hash(provider_code, course)
     {
-      "You must say whether you can sponsor visas" =>
+      "You must say whether candidates can get a visa sponsored" =>
```

Comment:
We don't really write error messages like this. If this is the error message for the question page, we'd write it like this: 'Select if candidates can get visa sponsorship'

Reply:
Yeah, I was a bit surprised to see this! I guess it doesn't say "Select..." because it’s not appearing on the page whether the form is, but out-of-context on a different page (the course page)? But maybe it's still ok to say "Select..."?

Reply:
Actually turns out that this line wasn't shown anywhere, see https://github.com/DFE-Digital/publish-teacher-training/pull/1890/files#r700971070

Have removed.